cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -458,16 +458,6 @@ define([
             AjaxHelpers.expectNoRequests(requests);
         });
 
-        it('should have appropriate class names on focus/blur', function () {
```

Comment:
I ended up deleting this whole test due to difficulty testing focus and blur after the JQuery update. The check on line 466 passed most of the time, but I saw two sporadic failures of it (once in Jenkins, once locally). The check on line 468 only worked in dev mode, not run (as is consistent with comments I've seen on JQuery's own PRs/forums).

JQuery has pretty much given up on testing focus in unit tests. I'm happy to leave focus tests that we have if they pass reliably, but I don't think it is worth the risk of a sporadic test failure in this case.